src/Microsoft.Deployment.DotNet.Releases/src/Microsoft.Deployment.DotNet.Releases.csproj

```diff
@@ -1,47 +1,36 @@
-<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
This library must support trimming so I don't think we can target netstandard

Reply:
It already target netstandard2.0. I just moved the property up here.

Reply:
This library already targets netstandard2.0 on main. This PR just moved the property up.